udata/templates/organization/sidebar-producer.html

```diff
@@ -14,6 +14,11 @@
 {% else %}
 <br/>
 {% endif %}
+<button type="button" class="btn btn-sm btn-block btn-left btn-warning" @click="$refs.discussions.start('')"
+    v-tooltip="'{{ _('Contact the producer') }}'" tooltip-placement="left">
+    <span class="fa fa-envelope-o"></span>
+    {{ _('Contact') }}
```

Comment:
Wondering about _Contact_ and _Contact the producer_.
In French, it renders "Contact" while it should be "Contacter" (as the next button says "Suivre" – present indicative). Is the translation missing or is it just a scope missing?